firefox-ios/firefox-ios-tests/Tests/ClientTests/Settings/SummarizeSettingsViewControllerTests.swift

```diff
@@ -32,10 +34,28 @@ final class SummarizeSettingsViewControllerTests: XCTestCase {
         XCTAssertEqual(sections.last?.children.count, 1)
     }
 
+    @MainActor
+    func test_generateSettings_withoutShakeFeature_hasExpectedSections() {
+        setupNimbusHostedSummarizerTesting(isEnabled: false)
+        let subject = createSubject()
+        let sections = subject.generateSettings()
+        XCTAssertEqual(sections.count, 1)
+        XCTAssertEqual(sections.first?.title, nil)
```

Comment:
Why the nil check here ? Also maybe `XCTAssertNil` is more semantic.

Reply:
thanks for the catch! me being lazy and copying over the other tests 🤣, updated in - https://github.com/mozilla-mobile/firefox-ios/pull/28766/commits/a1f024ba9970e04f21a609212d48a6d2d7fe175e

Reply:
also checking nil because we don't want the title to the first section and if we do, then it should be an intentional change and we add tests.